src/shared/containers/Status/style.scss

```diff
@@ -0,0 +1,66 @@
+@import '~styles/mixins';
+
+.container {
+  position: fixed;
+  display: flex;
+  background: $tc-gray-neutral-dark;
+  border-radius: 4px;
+  padding: $base-unit;
+  left: $base-unit;
+  bottom: $base-unit;
+  opacity: 0.7;
+  transition: opacity 200ms ease;
+
+  &:hover {
+    opacity: 1;
+  }
+
+  svg {
+    margin-left: $base-unit;
```

Comment:
well, in such cases you better make equal left and right margins (like half of the necessary spacing between the elements), then you won't have to set different margin for the first/last children of the container, and that will simplify the stylesheet and make easier its future modifications.